src/nupic/regions/tm_region.py

```diff
@@ -805,7 +805,22 @@ def writeToProto(self, proto):
       self.computePredictedActiveCellIndices)
     proto.orColumnOutputs = self.orColumnOutputs
 
-    self._tfdr.write(proto.temporalMemory)
+    if self.temporalImp == "py":
```

Comment:
Isn't that what htmpredictionmodel_serialization_test is for? The regions aren't easily tested outside a network.